dev_tools/commitizen/CHANGELOG.md.j2

```diff
@@ -0,0 +1,23 @@
+{# Jinja2 changelog template used by Commitizen #}
```

Comment:
Was this template referenced from somewhere? If so, consider adding a link if possible. This might be useful if/when there are changes to the Commitizen standard.